shell/utils/formatter.js

```diff
@@ -0,0 +1,12 @@
+
+import { SECRET_TYPES } from '@shell/config/secret';
+
+export function formatEncryptionSecretNames(secrets, chartNamespace) {
+  return secrets.filter(
+    (secret) => (secret.data || {})['encryption-provider-config.yaml'] &&
+        secret.metadata.namespace === chartNamespace &&
+        !secret.metadata?.state?.error &&
+        !!secret.metadata?.name &&
```

Comment:
Why the check for `metadata.name`? This should always be defined (though during secret creation `generateName` may be set instead). I'm sure this additional check wont break anything, just wondering why it was added

Reply:
@cnotv brought up a point that metadata is optional, so name might not exist

Reply:
I double-chcked the kubernetes docs, it's always gonna be defined https://kubernetes.io/docs/concepts/overview/working-with-objects/names/